ui/src/app/pages/artifactVersion/components/tabs/errorTab.tsx

```diff
@@ -0,0 +1,137 @@
+                                        data-testid="error-btn-details"
+                                        onClick={this.showDetails}>Show details</Button>
+                            </EmptyStateSecondaryActions>
+                        </EmptyState>
+                        <div className="separator">&nbsp;</div>
+                        {
+                            this.state.isShowDetails ?
+                                <div className="ace-wrapper pf-c-empty-state pf-m-lg" id="ace-wrapper">
+                                    <AceEditor
+                                        data-testid="ace-details"
+                                        mode="json"
+                                        theme="tomorrow"
+                                        name="errorDetail"
+                                        className="errorDetail"
+                                        width={this.state.editorWidth}
+                                        height={this.state.editorHeight}
+                                        fontSize={14}
+                                        showPrintMargin={false}
+                                        showGutter={false}
+                                        highlightActiveLine={false}
+                                        value={this.errorDetail()}
+                                        readOnly={true}
+                                        setOptions={{
+                                            enableBasicAutocompletion: false,
+                                            enableLiveAutocompletion: false,
+                                            enableSnippets: false,
+                                            showLineNumbers: true,
+                                            tabSize: 2,
+                                            useWorker: false
+                                        }}
+                                    />
+                                </div>
+                                :
+                                <div/>
+                        }
+            </div>
+        )
+    }
+    
+    protected initializeState(): ErrorTabContentState {
+        return {
+            editorHeight: "250px",
+            editorWidth: "100%",
+            isShowDetails: false
+        };
+    }
+
+    private errorMessage(): string {
+        if (this.props.error) {
+            return this.props.error.errorMessage;
+        } else {
+            return "Internal server error";
+        }
+    }
+
+    // Error description can be in node children - if it is missing, default description is used
+    private errorDescription(): ReactNode {
+        if (this.props.children) {
+            return this.props.children;
+        } else {
+            return (
+                "Something went seriously wrong on the server.  We'll try to provide you with some more information about " +
+                "the problem (see below)."
```

Comment:
I'd leave the word `seriously` from the error message, unless the server is on fire:)

Reply:
I'll update the text of this. :)